tests/Fixer/Alias/PowToExponentiationFixerTest.php

```diff
@@ -0,0 +1,228 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\Alias;
+
+use PhpCsFixer\Test\AbstractFixerTestCase;
+
+/**
+ * @author SpacePossum
+ *
+ * @internal
+ */
+final class PowToExponentiationFixerTest extends AbstractFixerTestCase
+{
+    /**
+     * @param string      $expected
+     * @param string|null $input
+     *
+     * @dataProvider provideFixCases
+     *
+     * @requires PHP 5.6
+     */
+    public function testFix($expected, $input = null)
+    {
+        $this->doTest($expected, $input);
+    }
+
+    public function provideFixCases()
+    {
+        return array(
+            array(
+                '<?php 1**2;',
+                '<?php pow(1,2);',
+            ),
+            array(
+                '<?php 1**2?>',
+                '<?php pow(1,2)?>',
+            ),
+            array(
+                '<?php 1.2**2.3;',
+                '<?php pow(1.2,2.3);',
+            ),
+            array(
+                '<?php echo -2** 3;',
+                '<?php echo pow(-2, 3);',
```

Comment:
I would reason the subtraction here is not an operation at all, but just the number `-2`